.github/workflows/linux.bash

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 set -exo pipefail
 
-export PATH="/usr/lib/ccache:/usr/lib64/ccache:$PATH"
+export PATH="/usr/lib/ccache/bin:/usr/lib/ccache:/usr/lib64/ccache:$PATH"
 export CCACHE_DIR=/icinga2/ccache
 export CTEST_OUTPUT_ON_FAILURE=1
 CMAKE_OPTS=''
 
 case "$DISTRO" in
+  alpine:*)
+    # Packages inspired by the Alpine package, just
+    # - LibreSSL instead of OpenSSL 3 and
+    # - no MariaDB or libpq as they depend on OpenSSL.
+    # https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/icinga2/APKBUILD
+    apk add bison boost-dev ccache cmake flex g++ libedit-dev libressl-dev ninja-build tzdata
+    ln -vs /usr/lib/ninja-build/bin/ninja /usr/local/bin/ninja
+
+    CMAKE_OPTS="-DUSE_SYSTEMD=OFF -DICINGA2_WITH_MYSQL=OFF -DICINGA2_WITH_PGSQL=OFF"
+
+    # This test fails due to some glibc/musl mismatch regarding timezone PST/PDT.
+    # - https://www.openwall.com/lists/musl/2024/03/05/2
+    # - https://gitlab.alpinelinux.org/alpine/aports/-/blob/b3ea02e2251451f9511086e1970f21eb640097f7/community/icinga2/disable-failing-tests.patch
+    sed -i '/icinga_legacytimeperiod\/dst$/d' /icinga2/test/CMakeLists.txt
```

Comment:
As I wrote that test case, I was curious to find out what's supposed to be wrong here. That reference to https://www.openwall.com/lists/musl/2024/03/05/2 makes no sense to me. Yes, the test case can use `TZ=PST8PDT`, but only if `_WIN32` is defined:

https://github.com/Icinga/icinga2/blob/b96dc39ea291e4d56695eb820441ae399e6aba77/test/icinga-legacytimeperiod.cpp#L55-L62

Reply:
I temporary switched the `ifdef` guard logic from `ifndef _WIN32` to `ifdef __GNUC__` (there does not seem to be a musl macro) to let musl use the Windows parts, but the tests keep failing. As I have not found any real documentation in this regard - next to some hand wavering like the linked post -, I went with excluding the test as my primary motivation was to test the LibreSSL build.

Reply:
> Can you please create a separate issue for this including the output of the failing test?

Isn't that issue basically the same as https://github.com/Icinga/icinga2/issues/9801? These test cases also keep falling on my Mac, but I didn't care enough to invest time in debugging and fixing it so far.

Reply:
@yhabteab: Thanks, I failed finding this one before and have now created #10288. Unsure if they are just similar or identical.

Reply:
Yonas has a point, but I doubt Mac uses musl. :)